packages/venia-concept/src/components/ProductOptions/option.js

```diff
@@ -54,6 +69,9 @@ class Option extends Component {
                     items={values}
                     onSelectionChange={handleSelectionChange}
                 />
+                <p className={classes.selection}>
+                    {selectedValue?`Selected ${label} : ${this.valuesMap.get(Array.from(selectedValue).pop())}`:''}
```

Comment:
I'm also a little worried about human readable label values. Will it be `Selected fashion_color` or is the values label already human readable?

Reply:
I have kept the map  as { attribute_code : store_label } So the text will be `Selected Fashion Color<attribute_label>: Burgundy<store_label>`